src/utils.js

```diff
@@ -143,9 +143,11 @@ async function loadConfig(loaderContext, config, postcssOptions) {
   if (!tsLoader) {
     const opts = { interopDefault: true };
 
-    const jiti = require("jiti")(__filename, opts);
+    const { createJiti } = require("jiti");
 
-    tsLoader = (filepath) => jiti(filepath);
+    const jiti = createJiti(__filename, opts);
+
+    tsLoader = (filepath) => jiti.import(filepath, { default: true });
```

Comment:
`default: true` is effectively a shorthand for `mod?.default ?? mod` ([docs](https://github.com/unjs/jiti?tab=readme-ov-file#programmatic)). It was required to get tests passing, specifically the `cts` and `mts` config tests. I'm not exactly sure why this is necessary - perhaps due to some changes in how `interopDefault` works in v2.